packages/core/src/browser/style/dialog.css

```diff
@@ -105,3 +105,25 @@
     font-size: var(--theia-ui-font-size1);
     display: block;
 }
+
+.theia-DialogNode {
```

Comment:
please for new classes use up-to-date conventions: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#css

old classes were introduced before we had an agreement and are not changed to avoid breaking clients

Reply:
thank you for letting me know about this, I have updated the code with the required changes.